vividus-to-xray-exporter/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="console">
+            <PatternLayout pattern="%highlight{%d [%t] %-5p %c - %m%n}" />
+        </Console>
+        <RollingFile name="file" fileName="output/logs/log4.log" filePattern="output/logs/log4j.%i.log">
```

Comment:
do you really need logging to file?

Reply:
Following common approach, I will remove